plugins/services/src/js/structs/Application.js

```diff
@@ -230,4 +230,8 @@ module.exports = class Application extends Service {
 
     return `${Config.rootUrl}/service/${serviceName}/`;
   }
+
+  isFolder() {
+    return Array.isArray(this.list);
```

Comment:
`Application` doesn't feature a `list` property only a `ServiceTree` is a folder and features a _list_ of services and service trees – in other words, this method will always return false.